server/src/routes/adminRoute.js

```diff
@@ -2,6 +2,7 @@ import express from 'express';
 import bodyParser from 'body-parser';
```

Comment:
**Repeated logic**
Why is the body parser here surely this just needs to be your entry file to parse all routes? You are also calling it in the request route

Reply:
i am looking into it please